tests/getEntities.js

```diff
@@ -0,0 +1,227 @@
+const expect = require('chai').expect;
+const parse = require('..').convert;
+const b_ = require('@bem/entity-name').create;
+
+it('should return an array', () => {
+    expect(parse({ block: 'button2' })).to.be.an('Array');
+});
+
+it('should return array of zero length if bemjson is empty', () => {
+    expect(parse({})).to.have.lengthOf(0);
+    expect(parse([])).to.have.lengthOf(0);
+});
+
+describe('immutable', () => {
```

Comment:
I mean when we extracting entities from BEMJSON, initital BEMJSON must be unchaged.
What is the best way to chech it?